src/killEvent.js

```diff
@@ -1,17 +1,14 @@
 function killEvent(target, name, descriptor) {
-  let oldHandler = target;
 
-  function eventCanceler(...args) {
-    args[0].stopPropagation();
-    args[0].preventDefault();
+  function eventCanceler(event) {
+    event.stopPropagation();
+    event.preventDefault();
 
-    oldHandler.apply(this, args);
+    return target.apply(this, arguments);
```

Comment:
Good point. We're not doing anything in particular with arguments rather than passing them to the wrapped function so I guess that's acceptable.